### PR TITLE
Normalize Op using fp32

### PR DESCRIPTION
**Pull request by bschifferer**
Normalize Op used fp64 and was not compatible with tensorflow.
I added out_dtypes as parameter to provide the user the option to use fp32 instead of fp64

using fp64 broke some unittests

### CODE REVIEW

**Comment by bschifferer** [comment]
rerun tests

**Comment by benfred** [comment]
rerun tests

**Review by benfred — APPROVED** [review]

**Comment by benfred** [comment]
I think the default dtype for our output should be float32 instead of float64 - but realize that's a bigger change than just allowing float32 like done here

**Comment by benfred** [comment]
rerun tests

**Review by mikemckiernan — COMMENTED** [review]
This is a nice update.

**Review by karlhigley — APPROVED** [review]
I agree with @mikemckiernan's comment re: the default. Optimistically approving assuming that change will have no impact on the functionality.

File: nvtabular/ops/normalize.py [hunk 1]
```diff
@@ -46,12 +46,17 @@ class Normalize(StatOperator):
         cont_features = CONTINUOUS_COLUMNS >> ops.Normalize()
         processor = nvtabular.Workflow(cont_features)
 
+    Parameters
+    -----------
+    out_dtype : str, default is problem-specific
+        dtype of output columns.
```
[the remark this answers is not in the document]
Comment (bschifferer) [reply nvtabular/ops/normalize.py:52]: Yes, you are right - I copied the statement from another op :)

**Comment by bschifferer** [comment]
As Jenkins didnt run in the beginning with my branch, @benfred created a new PR directly in the repository: https://github.com/NVIDIA-Merlin/NVTabular/pull/1597

I will close this one.

**Pull request closed without merging**